fuse/data/augmentor/augmentor_toolbox.py

```diff
@@ -452,3 +451,39 @@ def aug_op_batch_mix_up(aug_input: Tuple[Tensor, Tensor], factor: float) -> Tupl
     img = img * (1.0 - factor) + factor * img_mix_up
     labels = labels * (1.0 - factor) + factor * labels_mix_up
     return img, labels
+
+
+def aug_op_random_crop_and_resize(aug_input: Tensor,
+                                  out_size,
+                                  crop_size: float = 1.0,  # or optional - Tuple[float, float]
+                                  x_off: float = 1.0,
+                                  y_off: float = 1.0, 
+                                  z_off: float = 1.0) -> Tensor:
+    """
+    random crop a (3d) tensor and resize it to a given size
+    :param crop_size: float <= 1.0 - the fraction to crop from the original tensor for each dim
+    :param x_off: float <= 1.0 - the x-offset to take 
+    :param y_off:
+    :param z_off:
+    :param out_size: the size of the output tensor
+    :return: the output tensor
+    """
+    in_shape = aug_input.shape
+
+    if len(aug_input.shape) == 4:
```

Comment:
else not supported error?

Reply:
I added support for 2d tensor. Need your help on how to handle other dimensions (error?)